remme/tp/basic.py

```diff
@@ -205,3 +227,66 @@ def get_latest_block_info(context):
         LOGGER.debug(f'Block with number successfully loaded: {block_info.block_num + 1}')
 
         return block_info
+
+    def withdraw_fee(self, context, updated_state=None, fee=TRANSACTION_FEE):
+        from remme.tp.node_account import NodeAccountHandler
+        from remme.tp.account import AccountHandler
+        from remme.tp.consensus_account import ConsensusAccountHandler
+
+        if updated_state is None:
+            updated_state = {}
+
+        sender_address = self.get_fee_address()
+        if sender_address is None:
+            raise InvalidTransaction("Cannot withdraw fee: sender address not set")
+
+        payer_account_prefix = sender_address[:6]
+
+        payer_account_pb_selector = {
+            AccountHandler()._prefix: Account,
+            NodeAccountHandler()._prefix: NodeAccount,
+        }
+
+        try:
+            payer_account_pb_class = payer_account_pb_selector[payer_account_prefix]
+        except KeyError:
+            raise InvalidTransaction('Cannot withdraw fee: invalid account type')
+
+        consensus_address = ConsensusAccountHandler.CONSENSUS_ADDRESS
+
+        if all([
```

Comment:
Why not just that? Using `all` is appropriate when we deal with large lists of booleans like `all(map(lambda x: x % 2 == 0, some_list))`

```python
sender_address in updated_state and consensus_address in updated_state
```

Reply:
Does not matter, if it used with large elements or not. Here it just adds a quite bit readiness.